### PR TITLE
Task 2 branch test

### PR DESCRIPTION
**Pull request by andreideaconu-fd**

### CODE REVIEW

**Review by PufTo — COMMENTED** [review]
comentariu

File: test.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
-does it work like this??????
+does it work like this???
 TEST: first commit on first new branch
+
+PR testing
+test 2
```
Comment (PufTo) [inline test.txt:5]: aloo

File: test.txt [hunk 2]
```diff
@@ -1,2 +1,5 @@
-does it work like this??????
+does it work like this???
 TEST: first commit on first new branch
+
```
Comment (PufTo) [inline test.txt:3]: aloo2

File: test.txt (hunk 1 above)
Comment (andreideaconu-fd) [reply test.txt:5]: ba p/la

File: test.txt [hunk 3]
```diff
@@ -1,2 +1,5 @@
-does it work like this??????
+does it work like this???
```
Comment (PufTo) [inline test.txt:1]: anajsdhfa

**Review by PufTo — CHANGES_REQUESTED** [review]

**Review by PufTo — APPROVED** [review]
All lgtm

**Pull request merged**